app/dao/notifications_dao.py

```diff
@@ -384,13 +389,5 @@ def get_financial_year(year):
 
 
 def get_april_fools(year):
-    return datetime(
-        year, 4, 1, 0, 0, 0, 0,
-        pytz.timezone("Europe/London")
-    ).astimezone(pytz.utc)
-
-
-def get_bst_month(datetime):
-    return pytz.utc.localize(datetime).astimezone(
-        pytz.timezone("Europe/London")
-    ).strftime('%B')
+    return pytz.timezone('Europe/London').localize(datetime(year, 4, 1, 0, 0, 0)).astimezone(pytz.UTC).replace(
```

Comment:
i think this would benefit from a docstring/comment explaining why we need to jump through this hoop, and also splitting this out into two lines, eg 
```
april_start = pytz.timezone('Europe/London').localize(datetime(year, 4, 1, 0, 0, 0))
return april_start.astimezone(pytz.UTC).replace(tzinfo=None)
```